### PR TITLE
doc: update structure and adjust things in README.md

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

**1.Describe what this PR did**
This PR updates the structure in README.md and adjust pouch logo and so on.
We can feel the change of this file.

For user manual, we propose that we should divide into two parts: CLI manual and API munual, and more API related docs into `docs/api` directory.

**2.Does this pull request fix one issue?** 
NONE

**3.Describe how you did it**
NONE

**4.Describe how to verify it**
NONE

**5.Special notes for reviews**
NONE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +29,20 @@ Pouch can pack, deliver and run any application. It provides the enviroment for
...
+
+- **Security**: Designed to be secure by default. Include lots of security features, like hypervisor-based container technology, lxcfs, patched Linux kernel and so on.
+- **P2P distribution**: Easy to integrate with [Dragonfly](https://github.com/alibaba/dragonfly) which is a P2P-base container image distribution system to significantly improve image distribution efficiency.
+- **Rich container**: Besides the common way of container usage, also includes `rich container` mode which integrates more services, prestart hooks and more things inside container to guarantee container's running like usual.
+- **Kernel compatibility**: supports more OCI-compatible runtimes to support old kernels, like linux kernel 2.6.32+.
+- **Simple and easy to use**: Very few steps needed to setup Pouch. 
+
+## Architecture
+
+We describe Pouch's architecture from two dimensions: **ecosystem architecture**  which illustrates the place of Pouch in container ecosystem and **component architecture** which shows how we design decoupled components inside Pouch. For more details, please refer to file [architecture.md](docs/architecture.md).
```
[the remark this answers is not in the document]
Comment (allencloud) [reply README.md]: It is already `shows`. Maybe you are so tired that a cup of coffee is needed. 😄

File: README.md [hunk 2]
```diff
@@ -28,6 +29,20 @@ Pouch can pack, deliver and run any application. It provides the enviroment for
 
 The ambitious of Pouch is embracing container ecosystem and perfecting container standards [OCI(Open Container Initiative)](https://github.com/opencontainers). With the ability Pouch provides, people can spend less energy transforming applications to be cloud native. 
 
+## Features
+
+Pouch project has many important features listed below:
+
+- **Security**: Designed to be secure by default. Include lots of security features, like hypervisor-based container technology, lxcfs, patched Linux kernel and so on.
+- **P2P distribution**: Easy to integrate with [Dragonfly](https://github.com/alibaba/dragonfly) which is a P2P-base container image distribution system to significantly improve image distribution efficiency.
+- **Rich container**: Besides the common way of container usage, also includes `rich container` mode which integrates more services, prestart hooks and more things inside container to guarantee container's running like usual.
```
[the remark this answers is not in the document]
Comment (allencloud) [reply README.md]: Yes, what you said is correct. We will add a separate doc for this denifitely.

**Comment by allencloud** [comment]
Update majority change request, PTAL @Letty5411 
And thanks a lot for your careful review. 🌹

**Comment by Letty5411** [comment]
LGTM @allencloud

**Pull request merged**